manifests/charts/ztunnel/templates/daemonset.yaml

```diff
@@ -101,6 +101,8 @@ spec:
         {{- end }}
         - name: RUST_LOG
           value: {{ .Values.logLevel | quote }}
+        - name: RUST_BACKTRACE
```

Comment:
We won't get much from this on release builds (no debug symbols)?

Reply:
I think you still get pretty good info here, will test